images/bootstrap/runner

```diff
@@ -17,8 +17,22 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+# get test-infra for latest bootstrap etc
+git clone https://github.com/kubernetes/test-infra
+
+
+# Check if the job has opted-in to bazel remote caching and if so generate 
+# .bazelrc entries pointing to the remote cache
+export BAZEL_REMOTE_CACHE_ENABLED=${BAZEL_REMOTE_CACHE_ENABLED:-false}
```

Comment:
is it necessary to reexport this?

Reply:
it's useful for debug because we can see it in the env dumping from bootstrap, this pattern comes from the d-in-d snippet lower down